system-tests/projects/protocol/cypress/support/e2e-preload-service-worker.ts

```diff
@@ -0,0 +1,9 @@
+const setupServiceWorker = async () => {
+  try {
+    await window.navigator.serviceWorker?.register(new URL('http://localhost:2121/cypress/fixtures/service-worker-assets/scope/service-worker.js'))
+  } catch (e) {
+    // This errors the first time through before top is reloaded
```

Comment:
Can you set the `baseUrl` in the config so top doesn't need to be reloaded.

Reply:
There's some weirdness with the setup here. Leaving as-is for now.